docs/examples/KTabsList/AppearanceOverrides.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div>
```

Comment:
Can we remove the wrapping `div` since it's empty anyhow (has no styling or functionality associated with it)?